compiler-rt/lib/rtsan/rtsan_suppressions.cpp

```diff
@@ -72,12 +72,10 @@ bool __rtsan::IsStackTraceSuppressed(const StackTrace &stack) {
   if (!suppression_ctx->HasSuppressionType(call_stack_flag))
     return false;
 
-  Symbolizer *symbolizer = Symbolizer::GetOrInit();
-  Suppression *s;
-
   for (uptr i = 0; i < stack.size && stack.trace[i]; i++) {
     const uptr addr = stack.trace[i];
 
+    Symbolizer *symbolizer = Symbolizer::GetOrInit();
```

Comment:
I think this move made it worse? Now we unnecessarily call `GetOrInit` all over? I meant the Suppression only.

Reply:
Sorry, reading the comment in my email pointed me to the wrong line and I didn't engage critically. Thanks for double checking.